stories/atoms/components/listItem/ListItem.stories.mdx

```diff
@@ -62,27 +62,39 @@ The `ListItem` allows to display an element of a [List](/docs/atoms-list--overvi
         type: { summary: 'string | JSX.Element' }
       }
     },
-    leftElement: {
+    firstElement: {
       control: false,
       table: {
         type: { summary: 'JSX.Element' }
       }
     },
-    rightElement: {
+    lastElement: {
       control: false,
       table: {
         type: { summary: 'JSX.Element' }
       }
+    },
+    layout: {
+      description: 'The layout props from the List component.'
+    },
+    onClick: {
+      control: 'boolean'
```

Comment:
Yes maybe it's not worth having as it could be confusing to the user?

I added the control to be able to show the hover behaviour but as `onClick` is not a boolean it might be better to just omit the control and have the user explore the `onClick` story to see this.

What do you guys think @lolottetheclash @JoeyDjr ?

Reply:
Yes, please, remove the control as `onClick` is just a method.